src/components/UserPopup/index.tsx

```diff
@@ -42,9 +42,9 @@ export const UserPopup = (props: UserPopupProps) => {
     accountManagementUrl
   } = props;
 
-  function logoutFunction() {
-    logoutMethod ? logoutMethod() : keycloak!.logout();
-  }
+
+  }function logoutFunction() {
+   keycloak && (logoutMethod ? logoutMethod() : keycloak.logout())
```

Comment:
This is not the expected behavior. 
When someone implements the logoutMethod probably keycloak is not defined. Therefore it should not be required to call logoutMethod